### PR TITLE
Added isDone method to fileset.

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

**Comment by mickel1138** [comment]
Needs test...

File: src/Controller/UI/DatalandController.php [hunk 1]
```diff
@@ -90,6 +90,11 @@ public function defaultAction(string $udi)
                 $wkt = null;
             }
         }
+        
+        $fileset = $dataset->getDatasetSubmission()->getFileset();
+        
+        dd($fileset->isDone());
```
Comment (fightingtexasaggie) [inline src/Controller/UI/DatalandController.php:96]: debug code

Comment (mickel1138) [reply src/Controller/UI/DatalandController.php:96]: That file shouldn't be there...

File: src/Entity/Fileset.php [hunk 2]
```diff
@@ -64,4 +66,26 @@ public function removeFile(File $file)
...
+    public function isDone() :bool
+    {
+        $criteria = Criteria::create()
+        ->where(
+            new Comparison(
+                'status',
+                Comparison::IN,
+                array(
+                    File::FILE_NEW,
+                    File::FILE_IN_PROGRESS
```
Comment (praneethpr) [inline src/Entity/Fileset.php:84]: What about error? should we add that too?

Comment (mickel1138) [reply src/Entity/Fileset.php:84]: I think error means processed. We can add it later if FILE_ERROR has meaning. How does that sound?

**Review by praneethpr — APPROVED** [review]

**Review by fightingtexasaggie — APPROVED** [review]
looks pretty straightforward using criteria, test ran clean

**Pull request merged**